packages/react-core/src/components/Toolbar/__tests__/ToolbarToggleGroup.test.tsx

```diff
@@ -124,9 +124,10 @@ describe('Toolbar', () => {
         <ToolbarContent>{items}</ToolbarContent>
       </Toolbar>
     );
-    expect(view.find('#save-button').exists());
-    expect(view.find('#clear-button').exists());
-    expect(view).toMatchSnapshot();
+    // Expecting 2 matches for text because the buttons also exist in hidden expandable ontent for mobile view
```

Comment:
Nit: small typo

```suggestion
    // Expecting 2 matches for text because the buttons also exist in hidden expandable content for mobile view
```